tests/e2e/core-tests/specs/merchant/wp-admin-order-apply-coupon.test.js

```diff
@@ -50,7 +50,7 @@ const runOrderApplyCouponTest = () => {
 			// Verify the coupon list is showing
 			await page.waitForSelector('.wc-used-coupons');
 			await expect(page).toMatchElement('.wc_coupon_list', { text: 'Coupon(s)' });
-			await expect(page).toMatchElement('.wc_coupon_list li.code.editable', { text: couponCode });
+			await expect(page).toMatchElement('.wc_coupon_list li.code.editable', { text: 'code-fixed cart discount' });
```

Comment:
The updated version of `createCoupon` allow the changes to this file to work but the original version of this test should also pass with the new `createCoupon`. Can we revert this file to the original?

Reply:
Unfortunately, no. I changed this because it must look for a lowercased coupon name, only at this place when you apply a coupon it will be shown in all lowercases, so this is a specific place for the coupon. And, current createCoupon wouldn't work if lowercased coupon name (due to discountType names).

Reply:
> I changed this because it must look for a lowercased coupon name

`createCoupon()` should use `toLowerCase()` to return an accurate coupon code.

Reply:
Nice, fixing it now.